src/_common/firebase/firebase.service.ts

```diff
@@ -14,11 +14,6 @@ const firebaseConfig = {
 	measurementId: Environment.firebaseMeasurementId,
 };
 
-export function initFirebase() {
-	// Just calling this should initialize it and save it.
-	getFirebaseApp();
-}
```

Comment:
is this just not used in web or has been relocated?

Reply:
not used at all